gravatar-ui/build.gradle.kts

```diff
@@ -50,8 +51,13 @@ android {
     buildFeatures {
         compose = true
     }
-    composeOptions {
-        kotlinCompilerExtensionVersion = "1.5.8"
+    composeCompiler {
+        // Enable 'strong skipping'
+        // https://medium.com/androiddevelopers/jetpack-compose-strong-skipping-mode-explained-cbdb2aa4b900
+        enableStrongSkippingMode.set(true)
```

Comment:
🤔 I'm unsure if we want to enable this in the library. From what I've read, it is still a work in progress and not totally stable. In the `demo-app,` it's not a big deal, but in the library, we want it to be as stable as possible. 

Do you have a concrete reason to enable it here?

Reply:
Oh it's something highly encouraged to be enabled and thought to enable it. If you don't consider experimental features in the library module, then that is a good reason to not enable for the library module. I didn't know this. I'll disable it for the library module then. Thanks for the feedback.

Reply:
Removed strong skipping for `gravatar-ui` module with b2a397e

Reply:
Thanks!